src/client/.storybook/webpack.config.js

```diff
@@ -20,7 +20,7 @@ module.exports = (baseConfig, env, defaultConfig) => {
     system: path.resolve(__dirname, '../src', 'system'),
     shell: path.resolve(__dirname, '../src', 'shell')
   }
-  defaultConfig = require('../config-overrides')(defaultConfig, env)
+  // defaultConfig = require('../config-overrides')(defaultConfig, env)
```

Comment:
If you don't need this be brave and delete, it can always be reverted from a previous commit.  Commented lines are likely to get forgotten about and then nobody knows why it was commented and not removed.

Reply:
Yeah. I will delete it. There is no config-overrides anymore.